lib/modules/stats/country_statistics_api.rb

```diff
@@ -0,0 +1,156 @@
+class Stats::CountryStatisticsApi
+  STATISTICS_API = Rails.application.secrets[:country_statistics_api].freeze
+  BASE_URL = STATISTICS_API['url'].freeze
+  ENDPOINTS = {
+    representative: {
+      name: 'Representative',
+      field: STATISTICS_API['representative_field']
+    },
+    well_connected: {
+      name: 'Well connected',
+      field: STATISTICS_API['well_connected_field']
+    },
+    importance: {
+      name: 'Areas of importance for biodiversity',
+      field: STATISTICS_API['importance_field']
+    }
+  }.freeze
+
+  ERRORS = {
+    representative: """
```

Comment:
Not sure what `"""` this means...

Reply:
It's basically a way of creating a string in multiple lines.
This is just for readability, but it does the same thing as single "double quotes"